app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -493,9 +487,26 @@ class PatientRepository @Inject constructor(
         }
 
     return businessIdStream
-        .flatMap { businessId ->
-          Completable.fromAction { database.businessIdDao().save(listOf(businessId)) }
-              .toSingleDefault(businessId)
+        .flatMap { businessId -> saveBusinessId(businessId).toSingleDefault(businessId) }
+        .flatMap { businessId -> touchPatientRecord(patientUuid).toSingleDefault(businessId) }
+  }
+
+  private fun touchPatientRecord(patientUuid: UUID): Completable {
+    return Completable
+        .fromAction {
+          database
+              .patientDao()
+              .touch(
+                  patientUuid = patientUuid,
+                  newUpdatedAt = Instant.now(utcClock),
```

Comment:
We do not need to change the `updatedAt` of `Patient` model.
We can just update the `syncStatus` using `database.patientDao().updateSyncStatus`.

Reply:
Right you are. I've fixed it!